definitions/output/sample_data/pages_10k.js

```diff
@@ -0,0 +1,16 @@
+publish("pages_10k", {
+    type: "table",
+    schema: "sample_data",
+    bigquery: {
+        partitionBy: "date",
+        clusterBy: ["client", "is_root_page", "rank"],
+        requirePartitionFilter: true
```

Comment:
This does make it slightly awkward to use in some situations.

Specifically for the Web Almanac where we're supposed to use June 2024 data. So what I do is write this:

```sql
SELECT
  column1
FROM
  `httparchive.all.pages`
WHERE
  date = '2024-06-01' AND
  someotherclause
```

And then I comment out the table and the date to temporarily use the sample data:

```sql
SELECT
  column1
FROM
  --`httparchive.all.pages`
  `httparchive.sample_data.pages_1k`
WHERE
  --date = '2024-06-01' AND
  someotherclause
```

With this change I'll not only need to do that, but also have to change the date to the latest date (which is actually a moving target since the Web Almanac analysis goes on over several months).

Not the worst, and it's only because we're looking back multiple months for the Web Almanac which I admit is unusual.

Then again, the flip side of that argument is that forcing people to specify a `date` is good practice if they upgrade to the full dataset. But then again not everyone does that so maybe still more annoying to have to specify a date.

So I'm in two minds about whether this is needed...

Reply:
Got it. I looked into other consumers queries and several of them also comment out date and other filters when switching to sampled (following you? :) ).

Personally, I use rank filter or sampling instead of sampled tables (easier switching back and forth):
```sql
SELECT
  column1
FROM
  `httparchive.all.pages` -- TABLESAMPLE SYSTEM (1 PERCENT)
WHERE
  -- rank = 5000 AND
  date = '2024-06-01' AND
  someotherclause
```

It's great we have a requirement on the full tables, and if we forget to uncomment the filter - we are safe.

Let's keep it flexible, no requirement here.